pom.xml

```
@@ -22,9 +22,9 @@
 
   <properties>
     <!-- runtime dependencies -->
-    <folio-service-tools.version>4.1.0</folio-service-tools.version>
-    <folio-spring.version>8.2.0</folio-spring.version>
-    <spring-cloud-starter-openfeign.version>4.1.3</spring-cloud-starter-openfeign.version>
+    <folio-service-tools.version>5.0.0</folio-service-tools.version>
+    <folio-spring.version>9.0.0</folio-spring.version>
+    <spring-cloud-starter-openfeign.version>4.2.0</spring-cloud-starter-openfeign.version>
     <commons-collections4.version>4.4</commons-collections4.version>
     <mapstruct.version>1.6.2</mapstruct.version>
```

Should not downgrade the version shipped with folio-spring-base.
```suggestion
    <mapstruct.version>1.6.3</mapstruct.version>
```